api/db/migrations/20240425154143_add-pix-pro-sante.js

```diff
@@ -0,0 +1,23 @@
+const TABLE_NAME = 'complementary-certifications';
+const key = 'PRO_SANTE';
+
+const up = async function (knex) {
+  const exists = await knex(TABLE_NAME).select(1).where({ key }).first();
```

Comment:
Neatpick : Le first contient un select

```suggestion
  const exists = await knex(TABLE_NAME).first('id').where({ key });
```

ou bien

```suggestion
  const exists = await knex(TABLE_NAME).select(1).where({ key }).limit(1);
```

Reply:
Note : je serais + en faveur d'aller vers la gestion du 'onConflict'

Reply:
done